llvm-spirv/lib/SPIRV/SPIRVUtil.cpp

```diff
@@ -1720,6 +1720,9 @@ class OpenCLStdToSPIRVFriendlyIRMangleInfo : public BuiltinFuncMangleInfo {
 private:
   OCLExtOpKind ExtOpId;
   ArrayRef<Type *> ArgTys;
+#ifdef __clang__
+  [[maybe_unused]]
+#endif
```

Comment:
This change should be committed to https://github.com/KhronosGroup/SPIRV-LLVM-Translator/.

Reply:
@AlexeySotkin, @AlexeySachkov, can you take care of this please?

Reply:
I gave that a try: https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1004
Looks like LLVM_ATTRIBUTE_UNUSED is not working in some translator build scenarios. @AlexeySotkin, @AlexeySachkov, please advise.